compio/benches/fs.rs

```diff
@@ -287,6 +361,43 @@ fn write_tokio(b: &mut Bencher, (path, offsets, content): &(&Path, &[u64], &[u8]
     })
 }
 
+fn write_tokio_std(b: &mut Bencher, (path, offsets, content): &(&Path, &[u64], &[u8])) {
+    let runtime = tokio::runtime::Builder::new_current_thread()
+        .enable_all()
+        .build()
+        .unwrap();
+    b.to_async(&runtime).iter_custom(|iter| async move {
+        let file = Arc::new(std::fs::OpenOptions::new().write(true).open(path).unwrap());
+        let offsets = Arc::new(offsets.to_vec());
+        let content = Arc::new(content.to_vec());
+
+        let start = Instant::now();
+        for _i in 0..iter {
+            let file = file.clone();
+            let offsets = offsets.clone();
+            let content = content.clone();
+
+            tokio::task::spawn_blocking(move || {
+                for offset in offsets.iter() {
```

Comment:
In this function, all write operations are performed inside the same `spawn_blocking` closure. It should be as fast as `write_std`.